src/core/server/saved_objects/migrations/actions/index.ts

```diff
@@ -143,6 +148,7 @@ export interface ActionErrorTypeMap {
   documents_transform_failed: DocumentsTransformFailed;
   request_entity_too_large_exception: RequestEntityTooLargeException;
   unknown_docs_found: UnknownDocsFound;
+  unsupported_cluster_routing_allocation: UnsupportedClusterRoutingAllocation;
```

Comment:
I renamed the error type from 'cluster_routing_allocation_disabled` because the original name wasn't accurate anymore.